dedibox-network/ipv6/how-to/configure-ipv6-linux.mdx

```diff
@@ -17,127 +17,66 @@ categories:
   - You have an account and are logged into the [Dedibox console](https://console.online.net)
   - You have [created](/dedibox/dedicated-servers/how-to/order-a-server) a Dedibox dedicated server
   - You have [requested](/dedibox-network/ipv6/how-to/request-prefix) a /48 IPv6 prefix
-  - You have configured the [DHCPv6 client](/dedibox-network/ipv6/how-to/configure-dhcpv6)
 </Message>
 
-<Message type="important">
-  * Make sure you have configured the [DHCPv6 client](/dedibox-network/ipv6/how-to/configure-dhcpv6) on your system before setting up the network configuration.
-  * Make sure [IPv6 SLAAC](/dedibox-network/ipv6/how-to/) is enabled on servers supporting this feature.
-</Message>
-
-## How to configure IPv6 on Debian
-
-1. Open the network configuration file `/etc/network/interfaces` in a text editor and edit it as follows:
-    ```
-    auto eno1
-    iface eno1 inet6 static
-        address IPV6ADDRESS
-        netmask PREFIXLENGTH
-    ```
 
-    Remember to replace `eno1` with the proper internet interface name.
-2. Restart the network:
-    ```
-    sudo systemctl restart networking.service
-    ```
+`systemd-networkd` is the network manager of [systemd](https://en.wikipedia.org/wiki/Systemd). It is provided natively on most Linux distributions and can be used for the configuration of IPv6 with its DHCPv6 client feature.
```

Comment:
```suggestion
`systemd-networkd` is the network manager of [systemd](https://systemd.io/). It is provided natively on most Linux distributions and can be used for the configuration of IPv6 with its DHCPv6 client feature.
```

Reply:
Suggestion applied in local git. Link updated.